Quick/QuickSpec.m

```diff
@@ -101,7 +101,11 @@ + (SEL)addInstanceMethodForExample:(Example *)example {
         [example run];
     });
     const char *types = [[NSString stringWithFormat:@"%s%s%s", @encode(id), @encode(id), @encode(SEL)] UTF8String];
-    SEL selector = NSSelectorFromString(example.name.qck_selectorName);
+    NSString *selectorName = [NSString stringWithFormat:@"%@_%@_%ld",
+                              example.name.qck_selectorName,
+                              example.callsite.file,
+                              (long)example.callsite.line];
+    SEL selector = NSSelectorFromString(selectorName);
```

Comment:
@hffmnn Yes, I would agree that the new names are too long--I prefer the way they are now...

Is there another way we can fix #299, besides changing the test selector names? If this really is the only way, couldn't we *just* use the line number, since this bug only manifests within the same file?